guides/common/modules/ref_satellite-storage-requirements.adoc

```diff
@@ -28,6 +28,7 @@ ifdef::installing-satellite-server-disconnected[30 GB]
 |/var/log/ |10 MB |10 GB
 
 |/var/opt/rh/rh-postgresql12/lib/pgsql/ |100 MB |10 GB
```

Comment:
Perhaps this should be dropped in favor of the parent path?

Reply:
Sure, thank you for catching this!!!